src/Compilers/CSharp/Test/CommandLine/CommandLineTests.cs

```diff
@@ -1423,10 +1423,7 @@ public void ArgumentParsing()
             Assert.False(parsedArgs.SourceFiles.Any());
 
             parsedArgs = CSharpCommandLineParser.Script.Parse(new[] { "/langversion:?" }, WorkingDirectory, sdkDirectory);
-            parsedArgs.Errors.Verify(
-                // error CS2007: Unrecognized option: '/langversion:?'
-                Diagnostic(ErrorCode.ERR_BadSwitch).WithArguments("/langversion:?").WithLocation(1, 1)
-                );
+            parsedArgs.Errors.Verify();
```

Comment:
@jcouv I supposed this is sufficient. Do we need to assert anything else?

Reply:
The change I referred to *adds* something (we print more info), but this test shows that we removed an error. The test is not covering that particular piece of code.